Presentation.Web/Controllers/API/V1/StsOrganizationSynchronizationController.cs

```diff
@@ -118,6 +119,15 @@ public HttpResponseMessage UpdateConnection(Guid organizationId, [FromBody] Conn
                 .Match(FromOperationError, Ok);
         }
 
+        [HttpGet]
+        [Route("changelog/{numberOfLastChangeLogs}")]
```

Comment:
route should be `connection/change-log`
the "query" should be... a query parameter, not a resource id

name should be `numberOfChangeLogs`